doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -81,16 +81,16 @@ For the avoidance of doubt, the following is considered to be **out of scope**:
 <a name="1.5"></a>
 ## 1.5 Approach
 
-The approach taken in this Reference Architecture is to start simple (i.e. with a basic Kubernetes architecture), and then add detail and additional features/extensions as is required to meet the requirements of the Reference Model.
+The approach taken in this Reference Architecture is to start as simply as possible (i.e. with a basic Kubernetes architecture), and then add detail and additional features/extensions as is required to meet the requirements of the Reference Model and the needs of the CNTT community.
```

Comment:
Can we change ` and the needs of the CNTT community` to, ` and the functional and non-functional requirements of common cloud native network functions` to make it clear the requirements are driven by common need.